scripts/tasks/copy.js

```diff
@@ -1,10 +1,41 @@
+const path = require('path');
+
+function expandSourcePath(pattern) {
+  const requireResolveCwd = require('../require-resolve-cwd');
+
+  if (!pattern) {
+    return null;
```

Comment:
It'll throw something like this:

```
TypeError: glob pattern string required
```

Now, I don't think we can ever get into this situation really because pattern is passed in from a key of a json object. I suppose I'm trying to be too cautious here.